docs/guides/moodleapp/customisation.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Moodle App Customisation
+sidebar_position: 3
+tags:
+  - Moodle App
```

Comment:
It's great you've added this `Moodle App` tag. It would be great if you could edit the pre-existing Moodle App pages and add this label to them too (it can be done in a separate issue).